custom_model_runner/datarobot_drum/resource/components/Python/generic_predictor/generic_predictor.py

```diff
@@ -70,12 +72,18 @@ def configure(self, params):
             from datarobot_drum.drum.language_predictors.r_predictor.r_predictor import RPredictor
 
             self._predictor = RPredictor()
-        elif self._run_language == RunLanguage.OTHER and self._with_triton_server:
-            from datarobot_drum.drum.language_predictors.triton_predictor.triton_predictor import (
+        elif self._with_triton_server and self.cli_adapter.target_type == TargetType.UNSTRUCTURED:
```

Comment:
I would add triton/nemo + target type check in the args parser.

Reply:
added the `--gpu-predictor=triton|nemo|vllm` option.